### PR TITLE
add option to report raw count values instead of delta values

### PR DESCRIPTION
**Pull request by fyi-coursera**
Reporting delta values doesn't behave correctly if the application is decrementing the counter.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ The reporter provides a fine-grained configuration options through its builder t
                     .withJvmMetrics()
                     .withGlobalDimensions("Region=us-west-2", "Instance=stage")
                     .withZeroValuesSubmission()
+                    .withReportwithReportRawCountValue()
```
Comment (azagniotov) [inline README.md]: Typo. You want `.withReportRawCountValue()`

Comment (fyi-coursera) [reply README.md]: thanks

**Comment by fyi-coursera** [comment]
@azagniotov any updates on this PR?

**Comment by azagniotov** [comment]
Will release it today! Thank you

**Comment by azagniotov** [comment]
Released `v1.0.10` to Maven Central. Please allow a few hours for the new artifact to become live.

**Comment by fyi-coursera** [comment]
Thanks!

**Pull request merged**